tiny_dnn/util/serialization_functions.h

```diff
@@ -854,6 +874,14 @@ struct serialization_buddy {
     layer.serialize_prolog(ar);
     ar(cereal::make_nvp("in_size", layer.in_shape()[0]));
   }
+
+  template <class Archive>
+  static inline void serialize(Archive &ar, tiny_dnn::softplus_layer &layer) {
+    layer.serialize_prolog(ar);
+    ar(cereal::make_nvp("in_size", layer.in_shape()[0]),
+       cereal::make_nvp("beta", layer.beta_),
+       cereal::make_nvp("beta", layer.threshold_));
```

Comment:
@nishnik:
`[  FAILED  ] serialization.serialize_softplus` because of this line.  
Changing `beta` -> `threshold` should pass this test.